app/api/schema/discount_codes.py

```diff
@@ -153,6 +154,53 @@ def validate_quantity(self, data, original_data):
                 raise UnprocessableEntity({'pointer': '/data/attributes/tickets-number'},
                                           "tickets-number should be greater than max-quantity")
 
+    @validates_schema(pass_original=True)
+    def validate_value(self, data, original_data):
+        if 'id' in original_data['data']:
+            try:
+                discount_code = DiscountCode.query.filter_by(id=original_data['data']['id']).one()
+            except NoResultFound:
+                raise ObjectNotFound({'parameter': '{code}'}, "DiscountCode: not found")
+
+            if 'type' not in data:
+                data['type'] = discount_code.type
+
+            if 'value' not in data:
+                data['value'] = discount_code.value
+
+        if data['type'] == "percent":
+            for ticket in data['tickets']:
```

Comment:
`if data['tickets']` 
Use this, it should fix the error

Reply:
@uds5501 same error. I'll check what's going wrong

Reply:
try `if 'tickets' in data`?